azure_jumpstart_ag/manufacturing/bicep/clientVm/clientVm.bicep

```diff
@@ -16,6 +16,10 @@ param windowsOSVersion string = '2022-datacenter-g2'
 @description('Location for all resources')
 param location string = resourceGroup().location
 
+
+@description('Name of the storage account')
+param aioStorageAccountName string = 'aiostg${namingGuid}'
```

Comment:
Do we plan to create the storage account for the user?

Reply:
Yes, as this is a pre-req for ESA